ucm2/sof-hda-dsp/HiFi.conf

```diff
@@ -18,7 +18,7 @@ SectionDevice."Mic1" {
 			Condition {
 				Type String
```

Comment:
This must be changed to RegexMatch and Haystack should be renamed to String. See `If.hs_init` for the RegexMatch condition example.